style.css

```diff
@@ -341,10 +341,6 @@ textarea {
 }
 
 @media (min-width: 992px) {
```

Comment:
- [ ] OPTIONAL: I strongly suggest you kindly correct these project cards:

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/149470294-7bc79b50-3648-45f2-872c-0dbb3f3e0b8b.png) | ![image](https://user-images.githubusercontent.com/57421823/149470218-591354cb-8da7-4039-b7d9-5181fcbcf45a.png)

Yours are done well! but look a little tiny (the Figma's are bigger and take about 80% of the horizontal width).

- [ ] OPTIONAL: I strongly suggest you kindly add this margin:

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/149470657-940f91d2-b90c-4064-a935-e834bd2a0077.png) | ![image](https://user-images.githubusercontent.com/57421823/149470762-d8a2a2c3-fc13-4e67-8fcb-81e306ab7125.png)

That way, your project will look striking!! 💪 

- [ ] OPTIONAL:  I'm marking this as optional because it won't stop my approval of this project, but please kindly next time, use the **same** pull request when submitting additional review requests, instead of creating a new branch with a new pull request.  Happy coding!